.github/workflows/backport.yml

```diff
@@ -9,6 +9,7 @@ on:
       - unlabeled
       - labeled
       - closed
+      - converted_to_draft
```

Comment:
I think we would want `ready_for_review` for conversions _from_ draft

Reply:
Indeed!